src/test/emulators/storage/rules/manager.spec.ts

```diff
@@ -0,0 +1,89 @@
+import { expect } from "chai";
+import { tmpdir } from "os";
+import { v4 as uuidv4 } from "uuid";
+
+import { FirebaseError } from "../../../../error";
+import { StorageRulesFiles } from "../../fixtures";
+import { StorageRulesManager } from "../../../../emulator/storage/rules/manager";
+import { StorageRulesRuntime } from "../../../../emulator/storage/rules/runtime";
+import { Persistence } from "../../../../emulator/storage/persistence";
+import { RulesetOperationMethod } from "../../../../emulator/storage/rules/types";
+
+describe("Storage Rules Manager", function () {
```

Comment:
please use arrow functions

Reply:
Done.

Reply:
Jk, have to extend the timeout duration.